figures/sites.py

```diff
@@ -210,6 +210,23 @@ def get_user_ids_for_site(site):
     return user_ids
 
 
+def get_edly_users_for_site(site):
+    if figures.helpers.is_multisite():
```

Comment:
I dont think we need to add this check as our infrastructure is multisite

Reply:
this method checks if `FIGURES_IS_MULTISITE` is set to `True` or `False`. Currently, we have not set this env variable and default value is `False`. Therefore, kept the check for compatibility. Also, existing method has this check: 
https://github.com/edly-io/figures/blob/dbee051fa76d5578f7e86639a7784a696962f060/figures/sites.py#L214